### PR TITLE
Refactor - Filter Page and restaurant page

### PR DESCRIPTION
**Pull request by Melody-Le**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (tlkiong) [inline package.json]: Why need this file?

File: src/style.css [hunk 2]
```diff
@@ -84,8 +83,8 @@ a {
 
 .search-option-container,
 .filter-catogery-container {
-  height: 20rem;
-  overflow-y: scroll;
+  /* height: 20rem;
+  overflow-y: scroll; */
```
Comment (tlkiong) [inline src/style.css]: Remove if not required. Don't submit commented code like this unless you have a very good reason to

File: package.json (hunk 1 above)
Comment (Melody-Le) [reply package.json]: currently, I no need to have this file. I will remove it.
Last time I create it to use import / export module , need to put `  "type": "module" ` into package.json to let it work.
 But in current code,  I don't import/export code between files, so I should remove it.

File: src/style.css (hunk 2 above)
Comment (Melody-Le) [reply src/style.css]: Noted.

File: package.json (hunk 1 above)
Comment (tlkiong) [reply package.json]: Yes. Please remove. once remove, please resolve this conversation

Comment (Melody-Le) [reply package.json]: I have removed it.

**Pull request closed without merging**